packages/tendermint-rpc/src/rpcclients/httpclient.ts

```diff
@@ -65,29 +65,67 @@ export interface HttpEndpoint {
   readonly headers: Record<string, string>;
 }
 
+export interface HttpClientOptions {
+  dispatchInterval: number;
+}
+
+export const defaultHttpClientOptions: HttpClientOptions = { dispatchInterval: 0 };
```

Comment:
This runs a CPU burner loop, right?

I think something like 20ms should be a sane interval time as this delay is still small compared to the network roundtrip.

Reply:
oof yes nice catch